packages/vuetify/src/mixins/bootable/index.ts

```diff
@@ -47,8 +47,8 @@ export default Vue.extend<Vue & Toggleable>().extend({
   },
 
   methods: {
-    showLazyContent (content?: VNode[]): VNode[] | undefined {
-      return this.hasContent ? content : undefined
+    showLazyContent (content?: () => VNode[]): VNode[] {
+      return (this.hasContent && content) ? content() : [this.$createElement()]
```

Comment:
```suggestion
      return (this.hasContent && content) ? content() : undefined
```

Following the comment made here: https://github.com/vuetifyjs/vuetify/pull/8823/files#diff-a5bc4eebd33025bd247fef3ba12f27dbR164